sdcm/db_stats.py

```diff
@@ -606,6 +606,10 @@ def get_setup_details(self):
                 else:
                     setup_details[key] = value
 
+        setup_details["additional_setup_details"] = {}
+        for key, value in self.params.additional_data_for_elastic.items():
```

Comment:
I think we can use `get_ami_tags` function to get any information we need out of the AMIs
I think we can do that when report it needed, and not at the beginning of the test, saving it till the end of the test.
that small information gonna be kept on all tests for no reason.
also it would better if the logic is in one place for those things and less scattred